cmd/fleet/cron.go

```diff
@@ -338,7 +338,7 @@ func checkNVDVulnerabilities(
 		err := nvd.Sync(opts)
 		if err != nil {
 			errHandler(ctx, logger, "syncing vulnerability database", err)
-			return nil
+			// don't return, continue on ...
```

Comment:
I wonder what happens if Sync fails the first time it tries to download. Would other things break?

Reply:
Yup, the Software to CPE step will fail because it won't be able to find the sqlite database